### PR TITLE
Feature/input component

### PR DESCRIPTION
**Pull request by ericeladio**

### CODE REVIEW

File: src/components/UI/InputIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+              : check
+              ? 'border-green-500'
+              : 'border-gray-300'
+          } ${styles ?? ''}`,
+        )}
+      />
+      {check && (
+        <Checkbox
+          cssClasses={'absolute top-[8px] right-2'}
+          color={''}
```
Comment (cyberpolin) [inline src/components/UI/InputIcon.tsx]: can this be not present, or is a needed property?

Comment (cyberpolin) [reply src/components/UI/InputIcon.tsx]: If this is needed, please go ahead and mergue this with dev

File: src/components/UI/InputIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        />
+      )}
+      {error && (
+        <p className="text-red-500 text-sm mt-1 ml-2">{errorMessaje}</p>
+      )}
+    </div>
+  );
+};
+
+export default InputIcon;
```
Comment (cyberpolin) [inline src/components/UI/InputIcon.tsx]: The component should be just Input, it is not an Icon, or perhaps BaseInput...

File: src/components/UI/InputIcon.tsx (hunk 1 above)
Comment (ericeladio) [reply src/components/UI/InputIcon.tsx]: if this property is not present, the fill-green-500 class don't work

**Pull request merged**